optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/roster/EmployeeRosterViewPanel.java

```diff
@@ -94,6 +94,10 @@ protected void refreshTable() {
             @Override
             public void onSuccess(EmployeeRosterView employeeRosterView) {
                 EmployeeRosterViewPanel.this.employeeRosterView = employeeRosterView;
+                if (null == employeeRosterView) {
+                    solveStatus.setTextContent("You are using the wrong version of Wildfly. Please use Wildfly 10.1.0.Final");
```

Comment:
We need to have a decent error handler on the client side to show all kinds of errors in a clear, popup with a red icon. This is a good temporary workaround, so let's add a comment "// TODO use error handler"

We probably don't want to hardcode the wildfly version in the error message, but we do want to mention it indeed.

Reply:
There is an ErrorPopupPresenter in uberfire, but none in Errai. I can easily create one using DialogBox (which is a generic popup).

Reply:
+1 good idea